lib/public/app.css

```diff
@@ -673,3 +673,27 @@ label {
         grid-template-columns: repeat(3, 1fr);
     }
 }
+
+.x-scrollable {
+    overflow-x: auto;
+    white-space: nowrap;
+} 
+
+.x-scrollable .table.left-sticky thead tr th:first-child {
+    position: sticky;
+    top: 0;
+    left: 0;
+    z-index: 2;
+    background-color: var(--color-gray-light);
+}
+
+.x-scrollable .table.left-sticky tbody tr td:first-child {
+    position: sticky;
+    top: 0;
+    left: 0;
+    z-index: 2;
+}
+
```

Comment:
This is a hack, and I don't know if it is a good idea to propose it as is, because it will stay and they will soon expect more. We should really try to propose only stable and clean stuff.
For example, what will you do when they will want 2 columns to be sticky?
Also, we should not have something like `left-sticky` to set z-index, and even less a background color... this will lead to maintenance hell!
For me, a first step is to put horizontal scroll, and then we see when reworking table to put the sticky column. But we should not provide such hack.

Reply:
1. At first I did it with modification of `rows`, `cell` components then with a bit of refactoring it could be more elastic and be used for more than one frozen (sticky) column, there is need only to one sticky column for, I'd say, primary-key column, which is always leftmost one.
2. I think it's quite natural to set z-index with sticky in this case, because leftmost column must be always on top of everything on right, what's
![Screenshot 2024-02-26 at 16 27 07](https://github.com/AliceO2Group/Bookkeeping/assets/65893715/6a1c016d-a874-4057-8c31-0dc4bf3f73cb)
 not the case without setting z-index. 
3. I dared to bring background colour here because:
    - it's to preserve common (used everywhere) styling of table
    - without this leftmost column is a bit hollow (see screenshot below)
    - for me, without sticky column, that scrolling is not very useful because you don't know run it is while looking one some quality
![image](https://github.com/AliceO2Group/Bookkeeping/assets/65893715/ece8c110-b816-4b12-b02c-1e349bb3f95e)

![image](https://github.com/AliceO2Group/Bookkeeping/assets/65893715/bbd631e0-0356-4cb3-a38d-dd9d2a076e90)